test/queries/queries_test.jl

```diff
@@ -101,14 +101,6 @@ end
 
 @testset "Infer Vtrees Test" begin
 
-    little = load_logic_circuit(zoo_psdd_file("little_4var.psdd")) 
-    little_vtree = zoo_vtree("little_4var.vtree")
-
-    inferred = infer_vtree(little)::Vtree
```

Comment:
is there other tests for `infer_vtree` is was it removed too?

Reply:
if there isn't another test for `infer_vtree`, then we should add one, but not in the `io` tests which have a different purpose